src/containers/template-24-col-grid/template.less

```diff
@@ -0,0 +1,14 @@
+@import '~antd/dist/antd.less'; // Import Ant Design styles by less entry
+
+.grid-content-container {
+    padding: 24px;
+}
+
+
+
+
+.example-col {
+    border: 1px @primary-color solid;
+    height: 50px;
+    text-align: center;
+}
```

Comment:
newline

Reply:
Hmmm, shouldn't lint enforce this? Might have to add `.less` to file extensions in lint command?